src/main/java/dev/pgm/community/requests/commands/RequestCommands.java

```diff
@@ -1,68 +1,65 @@
 package dev.pgm.community.requests.commands;
 
-import static dev.pgm.community.utils.PGMUtils.parseMapText;
 import static net.kyori.adventure.text.Component.space;
 import static net.kyori.adventure.text.Component.text;
 import static net.kyori.adventure.text.Component.translatable;
-import static tc.oc.pgm.util.text.TemporalComponent.duration;
 
-import co.aikar.commands.annotation.CommandAlias;
-import co.aikar.commands.annotation.CommandCompletion;
-import co.aikar.commands.annotation.CommandPermission;
-import co.aikar.commands.annotation.Default;
-import co.aikar.commands.annotation.Dependency;
-import co.aikar.commands.annotation.Description;
-import co.aikar.commands.annotation.Optional;
-import co.aikar.commands.annotation.Subcommand;
-import co.aikar.commands.annotation.Syntax;
+import cloud.commandframework.annotations.Argument;
+import cloud.commandframework.annotations.CommandDescription;
+import cloud.commandframework.annotations.CommandMethod;
+import cloud.commandframework.annotations.CommandPermission;
+import dev.pgm.community.Community;
 import dev.pgm.community.CommunityCommand;
 import dev.pgm.community.CommunityPermissions;
 import dev.pgm.community.requests.MapCooldown;
-import dev.pgm.community.requests.RequestConfig;
-import dev.pgm.community.requests.RequestProfile;
 import dev.pgm.community.requests.feature.RequestFeature;
-import dev.pgm.community.users.feature.UsersFeature;
 import dev.pgm.community.utils.BroadcastUtils;
 import dev.pgm.community.utils.CommandAudience;
-import dev.pgm.community.utils.MessageUtils;
 import dev.pgm.community.utils.PaginatedComponentResults;
-import java.time.Duration;
 import java.time.Instant;
 import java.util.Comparator;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
+import javax.annotation.Syntax;
 import net.kyori.adventure.text.Component;
 import net.kyori.adventure.text.event.ClickEvent;
 import net.kyori.adventure.text.event.HoverEvent;
 import net.kyori.adventure.text.format.NamedTextColor;
 import net.kyori.adventure.text.format.TextDecoration;
 import org.bukkit.entity.Player;
 import tc.oc.pgm.api.map.MapInfo;
-import tc.oc.pgm.util.Audience;
 import tc.oc.pgm.util.named.MapNameStyle;
-import tc.oc.pgm.util.named.NameStyle;
 import tc.oc.pgm.util.text.TemporalComponent;
 import tc.oc.pgm.util.text.TextFormatter;
 
 public class RequestCommands extends CommunityCommand {
 
-  @Dependency private RequestFeature requests;
-  @Dependency private UsersFeature users;
+  private static final String REQUEST_CMD = "request|req";
+  private static final String REQUESTS_CMD = "requests|reqs";
+  private static final String COOLDOWNS_CMD = "mapcooldowns|mapcd|mapcooldown";
 
-  @CommandAlias("request|req")
-  @Description("Request a map")
+  private final RequestFeature requests;
+
+  public RequestCommands() {
+    this.requests = Community.get().getFeatures().getRequests();
+  }
+
+  @CommandMethod(REQUEST_CMD + " <map>")
```

Comment:
I feel like declaring the constants outside the annotation hurts readability more than it helps maintainability, i'd suggest you keep the whole constants, and if it's repeated many times, put it on a class-level `@CommandMethod` with all the sub-commands inside the class